### PR TITLE
Polish English docs of elementwise_add/sub/mul/div

### PR DESCRIPTION
**Pull request by danleifeng**
Polish English docs of elementwise_add/sub/mul/div.
Similar to #19946 
cn_docs pr: https://github.com/PaddlePaddle/FluidDoc/pull/1312

### CODE REVIEW

File: paddle/fluid/operators/elementwise/elementwise_add_op.cc [hunk 1]
```diff
@@ -20,6 +20,28 @@ limitations under the License. */
...
+  }
+
+  void AddInputY() override {
+    AddInput("Y",
+             "(Variable), Tensor or LoDTensor of any dimensions. Its dtype "
+             "should be int32, int64, float32, float64.");
+  }
+
+  std::string GetOpFuntionality() const override {
+    return "two tensors add element-wise and returns a new resulting tensor";
```
Comment (gongweibao) [inline paddle/fluid/operators/elementwise/elementwise_add_op.cc]: Add arguments element-wise.

File: paddle/fluid/operators/elementwise/elementwise_add_op.cc [hunk 2]
```diff
@@ -45,10 +67,10 @@ class ElementwiseAddDoubleGradDescMaker
 }  // namespace paddle
 
 REGISTER_ELEMWISE_GRAD_MAKER(elementwise_add, Add);
-REGISTER_ELEMWISE_EXPLICIT_OP_WITHOUT_GRAD(elementwise_add, "Add",
-                                           "Out = X + Y");
+REGISTER_ELEMWISE_EXPLICIT_OP_WITHOUT_GRAD(elementwise_add, Add);
```
Comment (gongweibao) [inline paddle/fluid/operators/elementwise/elementwise_add_op.cc]: Why delete  "Out = X + Y"?

File: paddle/fluid/operators/elementwise/elementwise_add_op.cc (hunk 1 above)
Comment (danleifeng) [reply paddle/fluid/operators/elementwise/elementwise_add_op.cc]: done.

**Comment by danleifeng** [comment]
image
image
image
image

**Review by gongweibao — APPROVED** [review]
LGTM

**Review by hutuxian — APPROVED** [review]

**Review by kuke — APPROVED** [review]

**Pull request merged**